s1/Program copy.cs

```diff
@@ -0,0 +1,117 @@
+﻿int n_rows = 10;
+int n_cols = 30;
```

Comment:
this is C++ naming convention. We don't use underscores to name local variables. 
Should be for example `rowCount` or `nRows`, although we try to be more explicit and not use just `n` for number

Reply:
These invariants cold also be `const int`